src/main/java/frc/robot/subsystems/MAXSwerveModule.java

```diff
@@ -4,9 +4,13 @@
 
 package frc.robot.subsystems;
 
+import com.revrobotics.CANSparkMax;
+
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class MAXSwerveModule extends SubsystemBase {
+  private final CANSparkMax m_driveSparkMax;
```

Comment:
Be sure to add a brief summary of what these variables are for (i.e. ``// Motors``)